onnxruntime/python/tools/quantization/notebooks/imagenet_v2/mobilenet.ipynb

```diff
@@ -322,9 +320,43 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "session_quant = onnxruntime.InferenceSession(\"mobilenet_v2.uint8.onnx\")\n",
+    "session_quant = onnxruntime.InferenceSession(\"mobilenet_v2_uint8.onnx\")\n",
     "run_sample(session_quant, 'cat.jpg', categories)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3 Convert the models to ort format."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This step is optional, we will convert the *mobilenet_v2_float.onnx* and *mobilenet_v2_uint8.onnx* to ORT format, to be used in mobile applications."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!{sys.executable} -m onnxruntime.tools.convert_onnx_models_to_ort --optimization_level basic ./"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please find the following converted models in the same directory,\n",
+    "* mobilenet_v2_float.ort\n",
+    "* mobilenet_v2_uint8.ort\n",
+    "\n",
+    "Please note, there are temporary ONNX model files generated by the quantization process, which are converted to ort format as well, please ignore these files."
+   ]
```

Comment:
These should be in a temp dir that is deleted after conversion so the user should never know. Is that not the case?

Reply:
The temp files are from quantization, not from convert to ort

Reply:
OK. Would be good to ask the quantization folks to cleanup their temp data.